projects/storefrontapp-e2e-cypress/package.json

```diff
@@ -14,6 +14,7 @@
     "cy:run:mobile:ci": "cypress run --config-file cypress.ci.1905.json --spec \"cypress/integration/mobile/**/*\"",
     "cy:run:regression": "cypress run --spec \"cypress/integration/regression/**/*\"",
     "cy:cds:run:vendor": "cypress run --spec \"cypress/integration/vendor/cds/**/*\"",
+    "cy:cds:run:ci:2005": "cypress run --config-file cypress.ci.2005.json --record --key $CYPRESS_KEY --tag \"2005,all\" --spec \"cypress/integration/!(vendor|b2b)/**/*.e2e-spec.ts,cypress/integration/vendor/cds/**/*.e2e-spec.ts\"",
```

Comment:
maybe you want to add a `cds` tag here?

Reply:
tag as in --tag?

target as in --spec? If so, it's to remove not run the complete vendor folder or b2b and run all the rest, and the next one is to include only cds e2e with the all others

Reply:
changing `all` to `all-cds` as discussed for tag

target question is because they run those scripts, but will remove cds start run ccv2 and 1905